src/components/Sidebar.jsx

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
You can remove this space here. and in other places where there is unnecessary space.